src/data/equiv/basic.lean

```diff
@@ -1754,6 +1754,45 @@ lemma of_bijective_apply_symm_apply {α β} (f : α → β) (hf : bijective f) (
   (of_bijective f hf).symm (f x) = x :=
 (of_bijective f hf).symm_apply_apply x
 
+section
+
+variables {α' β' : Type*} (e : perm α') {p : β' → Prop} [decidable_pred p]
+  (f : α' ≃ subtype p)
+
+/--
+Extend the domain of `e : equiv.perm α` to one that is over `β` via `f : α → subtype p`,
+where `p : β → Prop`, permuting only the `b : β` that satisfy `p b`.
+This can be used to extend the domain across a function `f : α → β`,
+keeping everything outside of `set.range f` is kept fixed. The `equiv` given by `f` can
```

Comment:
```suggestion
where `p : β → Prop`, permuting only the `b : β` that satisfy `p b`.

This can be used to extend the domain across a function `f : α → β`,
keeping everything outside of `set.range f` is kept fixed. For this use-case `equiv` given by `f` can
```
modulo some line-wrapping

Reply:
Typo fix of the above suggestion:
```suggestion
where `p : β → Prop`, permuting only the `b : β` that satisfy `p b`.
This can be used to extend the domain across a function `f : α → β`,
keeping everything outside of `set.range f` fixed. For this use-case `equiv` given by `f` can
```